.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,60 @@
+name: Babelfish for PostgreSQL Documentation
+description: Propose changes to Babelfish for PostgreSQL Documentation
+title: "[Documentation]: "
+labels: ["documentation", "untriaged"]
+assignees:
+  - gopinathpai
+  - nasbyj
+  - rcv-aws
+  - susanmdouglas-aws
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this documentation improvement.
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: Please enter your mail address for contacting in the case of needing more information.
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: documentation-scope
+    attributes:
+      label: What needs to be changed, updated or improve in the Babelfish Documentation?
+      description: Be explanatory, as detailed as you can. 
+      placeholder: Tell us what you have in mind!
+      value: "This is my feature request!"
+    validations:
+      required: true
+  - type: textarea
+    id: brief-desc-implementation
+    attributes:
+      label: If want to provide us a more details about how to implement.
+      description: Share with us if you have more details about the implementation.
+    validations:
+      required: false 
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: Select the version of the targeted documentation.
```

Comment:
How about:

Select the Babelfish documentation version to which the change applies.